### ISSUE TITLE
Accessibility - Step 8 Topic 2 - Screen is Missing Headings

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#62]
On screen Step 8 Topic 2 there are missing headings. The single item ordered list items should be heading level two (h2) instead of the ordered list item.

Please look on any other Step and Topic screens is this is still an issue on them.

**Comment by adriennerlong** [comment]
Thanks for pointing this out. We've sent guidance to ICF for labeling section headers so hopefully this will be addressed soon.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Heading level two (h2) have been added but are part of a one item un-ordered list. It's confusing to hear VoiceOver read heading level two (h2) text then that it's a heading level two (h2) and finish with it's an un-ordered list with one item.

**Comment by jfc3-dol** [comment]
Of course the heading issue applies to all three platforms (iOS, Android, and web).

**Comment by adriennerlong** [comment]
From ICF: Fixed

**Comment by jfc3-dol** [comment]
VoiceOver reads the heading level two (h2) items now. Going to remove the iOS label to remind me that we need to still check the web version along with the Android version once those are available to me.

**Comment by jfc3-dol** [comment]
TalkBack reads the heading level two (h2) items now. Going to remove the Android label to remind me that we need to still check the web version once those are available to me.

**Comment by jfc3-dol** [comment]
When reading the heading level two (h2) items in IE 11 it reads them as an unorderd list item and not as a heading level two (h2) item.

Can we remove the unnecessary list item/unordered list for the heading level two (h2) items so these are read correctly. Also, do so for any other pages that has similar code.

**Comment by sanganalakshay** [comment]
Fixed

**Comment by jfc3-dol** [comment]
JAWS is not reading any of the headings on the individual Steps pages besides the heading level one (h1) and the heading level two (h2) "Topics". The one JAWS is skipping is the "Learning Objectives', "Key Terms", and "Further Resources" which are supposed to be heading level two (h2) items. My guess is the with them being toggles to show and hide content JAWS is confused.

**Comment by sanganalakshay** [comment]
Fixed
